app/(company)/declare/components/DeclarePage.js

```diff
@@ -0,0 +1,132 @@
+'use client';
+import MaxWidth from '@shared/layouts/MaxWidth';
+import Image from 'next/image';
+import SignModal from './SignModal';
+import gpApi from 'gpApi';
+import gpFetch from 'gpApi/gpFetch';
+import { Libre_Baskerville, Tangerine } from '@next/font/google';
+import { useEffect, useState } from 'react';
+
+const baskerville = Libre_Baskerville({
+  weight: ['400', '700'],
+  subsets: ['latin'],
+});
+
+const tangerine = Tangerine({
+  weight: ['400', '700'],
+  subsets: ['latin'],
+});
+
+export default function DeclarePage() {
+  const [signatures, setSignatures] = useState('');
+
+  const fetchSignatures = async () => {
+    try {
+      const api = gpApi.homepage.declarationSignatures.list;
+      const signers = await gpFetch(api, false, 3600);
+      setSignatures(signers.signatures);
+    } catch (e) {
+      console.log('error at fetchSignatures', e);
+      return {};
+    }
+  };
+
+  useEffect(() => {
+    fetchSignatures();
+  }, []);
+
+  return (
+    <MaxWidth>
+      <div className="xl:max-w-5xl mx-auto w-full items-center">
+        <div className="flex flex-col mt-12 mb-12">
+          <div>
+            <div className="flex flex-row ml-14 md:ml-0 justify-center mx-auto mb-6">
+              <Image
+                src="/images/heart.svg"
+                alt="GP"
+                width={56}
+                height={56}
+                className="mr-3"
+              />
+              <p className="text-4xl font-bold">
+                GOOD PARTY Declaration of Independence
+              </p>
+            </div>
+            <div className="flex flex-col justify-center items-center">
+              <div className={`md:w-[55%] text-sm ${baskerville.className}`}>
```

Comment:
move all the text to an array and iterate over it. this way if you have to change a style or anything like that you will do it in one place.